report-flat/README.md

```diff
@@ -0,0 +1,27 @@
+# report-flat
+
+> aet-printable-report
+
+## Build Setup
+
+``` bash
+# install dependencies
+npm install
+
+# serve with hot reload at localhost:8080
+npm run dev
+
+# build for production with minification
+npm run build
+
+# run unit tests
+npm run unit
```

Comment:
Do we plan to have some unit tests or this is leftover boilerplate from https://github.com/vuejs-templates/webpack/tree/master/template ?

Can we limit boilerplate to what we are actually using?